README.md

```diff
@@ -206,7 +155,10 @@ To avoid having to manually install your plugin, build and deploy your plugin us
 
 If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. After configuring it, just run:
 
+## License
 
 ## Security vulnerability disclosure
 
 Please report any security vulnerability to [https://mattermost.com/security-vulnerability-report/](https://mattermost.com/security-vulnerability-report/).
+
+## Get Help
```

Comment:
Maybe link to https://community.mattermost.com/core/channels/plugin-circleci?

Reply:
I shall do that